pinot-tools/src/main/java/org/apache/pinot/tools/tuner/meta/manager/collector/ColStatsAccumulatorObj.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.tools.tuner.meta.manager.collector;
+
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.pinot.tools.tuner.meta.manager.MetaManager;
+import org.apache.pinot.tools.tuner.strategy.AbstractAccumulator;
+
+
+/**
+ * Accumulator for relevant fields in metadata.properties and index_map
+ */
+public class ColStatsAccumulatorObj extends AbstractAccumulator {
+
+  Map<String, BigInteger> accumulatedStats = new HashMap<>();
+  Map<String, Map<String, String>> segmentStats = new HashMap<>();
+
+  private static final String CARDINALITY = "cardinality";
+  private static final String TOTAL_DOCS = "totalDocs";
+  private static final String TOTAL_NUMBER_OF_ENTRIES = "totalNumberOfEntries";
+  private static final String IS_SORTED = "isSorted";
+  private static final String INVERTED_INDEX_SIZE = "invertedIndexSize";
+
+  @Override
+  public String toString() {
+    return "ColStatsAccumulatorObj{" + "accumulatedStats=" + accumulatedStats + ", segmentStats=" + segmentStats + '}';
+  }
+
+  private String _segmentName;
+  private String _cardinality;
+  private String _totalDocs;
+  private String _totalNumberOfEntries;
+  private String _isSorted;
+  private String _invertedIndexSize;
+
+  public Map<String, BigInteger> getAccumulatedStats() {
+    return accumulatedStats;
+  }
+
+  public Map<String, Map<String, String>> getSegmentStats() {
+    return segmentStats;
+  }
+
+  public ColStatsAccumulatorObj addInvertedIndexSize(String invertedIndexSize) {
```

Comment:
these methods are not really adding the numbers, but instead just setting the values to the fields. Why do we need these? these should be params to the merge method right?